src/components/Paginator/Paginator.tsx

```diff
@@ -311,10 +318,10 @@ class Paginator extends React.Component<IPaginatorProps, IPaginatorState> {
 				</Button>
 				<TextField
 					lazyLevel={100}
-					{...textFieldProps}
 					onBlur={this.handleTextFieldChange}
 					onSubmit={this.handleTextFieldChange}
-					isDisabled={isDisabled}
+					isDisabled={isDisabled || textFieldProps.isDisabled}
+					{...textFieldProps}
```

Comment:
this change of order would result in textFieldProps.isDisabled taking precedence over isDisabled.
if not intentional change, would leave the spread operator on the line above

Reply:
Good callout. Updated and added a test case as an example for this.